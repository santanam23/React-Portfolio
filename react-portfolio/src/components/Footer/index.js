import React from 'react';

function Footer() {
    return (
        <footer className="footer">
        <h2>Made with love by Miriam Santana</h2>
        <p>
            &copy; 2021 Miriam Santana's Wed Development Inc.
        </p>
        </footer>
    );
}

export default Footer;