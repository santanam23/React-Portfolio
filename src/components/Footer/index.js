import React from "react";

function Footer() {
  return (
    <footer  style={{background:"teal", color:'white', padding:'10px', display:"flex", justifyContent:'space-around', }}>
      <h6>Made with love by Miriam Santana</h6>
      <p>&copy; 2021 Miriam Santana's Wed Development Inc.</p>
    </footer>
  );
}

export default Footer;
