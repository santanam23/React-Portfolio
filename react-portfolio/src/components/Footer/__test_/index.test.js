import React from 'react';
import { render, cleanup } from '@testing-library/react';
import '@testing-library/jest-dom/extend-expect';
import Footer from '..';

afterEach(cleanup)

describe('Footer component', () => {
    // First Test
    it('renders', () => {
      render(<Footer />);
    });
    it('matches snapshot DOM node structure', () => {
      // render Footer
      const { asFragment } = render(<Footer />);
      expect(asFragment()).toMatchSnapshot();
    })
})