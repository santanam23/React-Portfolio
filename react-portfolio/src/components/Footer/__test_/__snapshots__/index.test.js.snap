// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Footer component matches snapshot DOM node structure 1`] = `
<DocumentFragment>
  <footer
    class="footer"
  >
    <h2>
      Made with love by Miriam Santana
    </h2>
    <p>
      Â© 2021 Miriam Santana's Wed Development Inc.
    </p>
  </footer>
</DocumentFragment>
`;
